pkg/metricprovider/external_metric_provider.go

```diff
@@ -49,11 +49,13 @@ func (p *ExternalMetricProvider) GetExternalMetric(ctx context.Context, namespac
 	}
 	// Find the cron metric scaler
 	var cronScaler *CronScaler
+outer:
```

Comment:
We can use a function to do this, if find return the cron.

Reply:
done